### PR TITLE
fix: bump git version to 2.26.2

### PR DESCRIPTION
**Pull request by zhujian7**
- bump alpine from 3.8 to 3.12
- bump git from 2.18 to 2.26

In order to fix
- https://github.com/git/git/security/advisories/GHSA-hjc9-x69f-jqj7
- https://github.com/git/git/security/advisories/GHSA-qm7j-c969-7j4q
- filter sensitive information printing when git fetch/clone failed

**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @hyy0322 @supereagle @bbbmj

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: build/base/alpine/Dockerfile.arm64 [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM arm64v8/alpine:3.8
+FROM arm64v8/alpine:3.12
```
Comment (hyy0322) [inline build/base/alpine/Dockerfile.arm64:1]: arm64 alpine 支持 3.12 了吗

Comment (zhujian7) [reply build/base/alpine/Dockerfile.arm64:1]: yes

**Comment by hyy0322** [comment]
LGTM

**Pull request merged**